include/basin/player.h

```diff
@@ -20,6 +13,7 @@ struct player {
     struct connection* conn;
     struct world* world;
     struct entity* entity;
+    uint32_t protocol_version; 
```

Comment:
This was intentionally removed as it belongs in the `connection` struct.

Reply:
I had to re-add it, because there was an exception that was being thrown, (`player->protocol_version = conn->protocol_version`) due to the fact that the `player` structure did not have that variable, so I assumed that it would be better to re-add it.

Reply:
This could definitely work as a temporary fix to the problem I'm talking about, unless if we define `player->protocol_version` within the source file, rather than the header file.